libdnf/conf/config_main.cpp

```diff
@@ -187,8 +187,7 @@ class ConfigMain::Impl {
     OptionBool debug_solver{false};
     OptionStringList installonlypkgs{INSTALLONLYPKGS};
     OptionStringList group_package_types{GROUP_PACKAGE_TYPES};
-    OptionStringSet optional_metadata_types{
-        OptionStringSet::ValueType{libdnf::METADATA_TYPE_COMPS, libdnf::METADATA_TYPE_UPDATEINFO}};
+    OptionStringSet optional_metadata_types{OptionStringSet::ValueType{}};
```

Comment:
We can modify in dnf5 default but we should not modify library defaults. This default is insecure. I would rather investigate the cause then disabling loading.

Reply:
I added the blocker only because of the change of default type for download.